fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/ReindexResourceDAO.java

```diff
@@ -294,7 +294,9 @@ public void updateParameters(String tablePrefix, List<ExtractedParameterValue> p
 
         if (parameters != null) {
             JDBCIdentityCache identityCache = new JDBCIdentityCacheImpl(getCache(), this, parameterDao, getResourceReferenceDAO());
-            try (ParameterVisitorBatchDAO pvd = new ParameterVisitorBatchDAO(connection, null, tablePrefix, false, logicalResourceId, 100,
+            // Check if this is multitenant
+            boolean mtId = this.getFlavor().isMultitenant();
```

Comment:
Minor. `mtId` is usually the tenant id value (from fhir_admin.tenants). Perhaps isMultitenant would be a better name.